src/components/ButtonGroupPicker/styled/buttonGroup.js

```diff
@@ -18,6 +19,18 @@ const StyledButtonGroup = styled.div`
     > label:only-child > span {
         border-radius: 100px;
     }
+
+    ${props =>
+        props.variant === 'shaded' &&
+        `
+        box-shadow:${props.shadows.shadow_10};
+        border: 1px solid transparent;
+        background-color: ${props.theme.rainbow.palette.background.main};
```

Comment:
thanks to `attachThemeAttrs` here you can use `props.palette.background.main` as you did with shadow below,
also this will cause the component to break when `props.theme.rainbow` does not exist which occurs when component is used in isolation without the Application